commentsComponent/server/index.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+
+const app = express();
+const PORT = 3001;
+
+
+app.use(express.json());
+
+app.use('/comments', (req, res) => {
+//use express.static(path.join here? instead of a (req, res))
+//see groceryList
+});
+
```

Comment:
Looks good so far. I would put the express.static method above your use method at line 9 so that it immediately sends out your dist directory files.